scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e
+
+randomhex() {
+	local size="$1"
+	if [ -z "${size}" ]; then
+		size=32
+	fi
+	local val=$(hexdump -e '4/4 "%08x"' -n${size} /dev/random)
+	echo ${val}
+}
+
+if [ "$NEWCERT" = "1" -o ! -e /srv/cert.pem ]; then
```

Comment:
Nit: use `-s` instead of `-e` to also generate certificate if file exists but is empty.